index.html

```diff
@@ -345,6 +345,14 @@ <h2>
       which are addressed to ensure the protection of user data during the
       request process.
       </li>
+      <li>Implementation of credential managers, specifically in the role of [=holder=] software (commonly known as "digital wallets"),
+      including how they securely store or manage [=digital credentials=] or
+      advertise capabilities to [=digital credential/presentation|present=] or
+      [=digital credential/issuance|issue=] them to the [=user agent=], is out
+      of scope. The only exception is the transmission of [=digital
+      credential/issuance request data=] and [=digital credential/request
+      data=] to and from such software.
```

Comment:
This --
```
      of scope. The only exception is the transmission of [=digital
      credential/issuance request data=] and [=digital credential/request
      data|credential request data=] to and from such software.
```
-- will render (in part; these links have no href) as --
> The only exception is the transmission of [issuance request data]() and [request data]() to and from such software.

-- which reads weirdly to me.

I think the smallest change that will solve the problems I see would be to change the latter displayed `request data` to read `credential request data`. I *think* my suggested change below does that, but I'm not natively fluent in the `[= =]` syntax, so please review and apply with care.
```suggestion
      of scope. The only exception is the transmission of [=digital
      credential/issuance request data=] and [=digital credential/request
      data=] to and from such software.
```